trigger.sh

```diff
@@ -0,0 +1,172 @@
+# This is main script to trigger edgex-taf tests in non-docker mode in pipeline
```

Comment:
@rajatkumar-eaton as of now it will overwrite the same. because we are assuming that every time in pipeline we are cloning new edgex-taf. so changes will not be there